src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,190 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * A personal assistant chatbot.
+ */
+public class Duke {
+    private static final String INDENTED_HORIZONTAL_LINE = " ".repeat(4) + "_".repeat(60);
+    /** Platform independent line separator */
+    private static final String LS = System.lineSeparator();
+
+    private static final Scanner SCANNER = new Scanner(System.in);
```

Comment:
Suggest to not use all-caps style on this variable name, for it is not a constant.

Reply:
Could I clarify, what do you mean by "not a constant"? I thought it was a constant because they are not changed once it is defined. I was also referring to the Contacts repository (https://github.com/nus-cs2113-AY2122S1/contacts/blob/master/src/main/java/Contacts1.java#L24 and https://github.com/nus-cs2113-AY2122S1/contacts/blob/master/src/main/java/Contacts1.java#L107)